src/EnergyPlus/WaterThermalTanks.cc

```diff
@@ -10639,6 +10639,96 @@ namespace WaterThermalTanks {
         return PLRResidualHPWH;
     }
 
+    bool SourceHeatNeed(int const WaterThermalTankNum,
+                        Real64 const OutletTemp,
+                        Real64 const DeadBandTemp,
+                        Real64 const SetPointTemp
+        ){
+        // FUNCTION INFORMATION:
+        //       AUTHOR         Yueyue Zhou
+        //       DATE WRITTEN   May 2019
+        //       MODIFIED       na
+        //       RE-ENGINEERED  na
+
+        // PURPOSE OF THIS FUNCTION:
+        // Determine by tank type, tank temperature and control mode if source side flow is needed
+
+        // METHODOLOGY EMPLOYED:
```

Comment:
If you aren't going to actually include a methodology, just eliminate these placeholders/stubs.